src/components/Geogebra/GeogebraViews.js

```diff
@@ -90,18 +90,20 @@ export default class {
             }
         });
 
-        api.service('workshops').on('xml-changed', (workshop) => {
+        /* api.service('workshops').on('xml-changed', (workshop) => {
```

Comment:
Why do you comment this out?

Reply:
Loading xml into admin views changes their sizes, it looks ugly. Setting elements should be enough. There's an open question how to handle perspectives in admin views.